File: app/src/main/java/com/adsamcik/signalcollector/data/StatData.java
package com.adsamcik.signalcollector.data;

import com.vimeo.stag.UseStag;

@UseStag
public class StatData {
	public final String id;
	public final String value;

	StatData(String id, String value) {
		this.id = id;
		this.value = value;
	}
}

File: statistics/src/main/java/com/adsamcik/tracker/statistics/database/data/StatData.kt
package com.adsamcik.tracker.statistics.database.data

import androidx.room.ColumnInfo
import androidx.room.Entity

/**
 * Database entity for statistics.
 */
@Entity(
		tableName = "statCache",
		primaryKeys = ["session_id", "cache_id"]
)
data class StatData(
		/**
		 * Session id
		 */
		@ColumnInfo(name = "session_id")
		val sessionId: Long,
		/**
		 * Cache id
		 */
		@ColumnInfo(name = "cache_id")
		val cacheId: String,
		/**
		 * Cache value
		 */
		val value: String
)
